src/include/network/lan/complex.rb

```diff
@@ -443,26 +443,24 @@ def handleOverview(_key, event)
             end
           end
 
-          return :edit
-        when :delete
-
-          # warn user when device to delete has STARTMODE=nfsroot (bnc#433867)
-          if NetworkInterfaces.GetValue(
-            Ops.get_string(LanItems.getCurrentItem, "ifcfg", ""),
-            "STARTMODE"
-            ) == "nfsroot"
-            if !Popup.YesNoHeadline(
-              Label.WarningMsg,
-              _("Device you select has STARTMODE=nfsroot. Really delete?")
-              )
-              # y2r: cannot break from middle of switch
-              # but in this function return will do
-              return nil
+            return :edit
```

Comment:
yes it is. Indentation is fixed in SLE-12 branch but not in SP1. So it looks weird. I expect rubocop will complain, so I'll fix it with separate commit.